projects/plugins/jetpack/modules/google-fonts.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Module Name: Google Fonts
+ * Module Description: A selection of Google fonts for Block enabled themes.
+ * Sort Order: 1
+ * Recommendation Order: 2
+ * First Introduced: $$next-version$$
+ * Requires Connection: No
+ * Auto Activate: Yes
```

Comment:
I don't know that I would auto-activate the module until we have a UI for it in the Jetpack > Dashboard, so folks can disable the feature if they'd like it.

Reply:
Changed to `No`. Do you know how we could auto-activate it only on Atomic sites?

Let me know if there are any other module header changes you would recommend. I wasn't sure how to set things like recommendation order.

Reply:
There are different ways we could go about it, depending on how you'd like to approach the deployment of this feature. Here is what comes to mind:

- If you'd rather not offer any UI, and enable it on all WoA sites right away, we could get rid of this module file in Jetpack, and instead require and instantiate the package from wpcomsh. This is what we do for the post-list package for example; the package isn't used in the Jetpack plugin, but is used in the wpcomsh plugin.
- If you'd like to offer a basic UI (in the legacy full module list) that would eventually turn into a proper setting in the Jetpack dashboard once the feature is deemed ready, you could enable the module on WoA like we do for the masterbar feature in wpcomsh. Check the `feature-plugins` directory for an example.

Reply:
I think it'd be nice to have a minimal UI for the module in Jetpack. I imagine lot's of plugin users will be interested in this feature, outside of WP.com hosted sites.

If you're willing, I'd like to get this basic version merged without a UI, and follow up with auto-activating the module for Atomic sites (wpcomsh) and adding a basic UI (Jetpack).